core/templates/a_hash_map.h

```diff
@@ -542,7 +535,7 @@ class AHashMap {
 		return Iterator(_elements + _size - 1, _elements, _elements + _size);
 	}
 
-	Iterator find(const TKey &p_key) {
+	_FORCE_INLINE_ Iterator find(const TKey &p_key) {
```

Comment:
This should not be force-inlined. Force-inlining long-ish functions like this may actually reduce performance in some situations because of code caches.

Reply:
Fair. I tested this over #112039 (which has `find()` all over a single function) and was able to see a performance reduction.

Will revert.